libraries/holochain-service-client/.gitignore

```diff
@@ -1,7 +1,6 @@
 build
 .tauri
 .gradle
-gradle
```

Comment:
Include gradle dir in repo which includes the config file for specifying which version to run via the gradle wrapper